WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -1,7 +1,14 @@
+import Combine
 import SwiftUI
+import class Yosemite.PointOfSaleOrderService
+import protocol Yosemite.PointOfSaleOrderServiceProtocol
+import struct Yosemite.PointOfSaleOrder
+import struct Yosemite.PointOfSaleCartItem
+import struct Yosemite.PointOfSaleCartProduct
 import protocol Yosemite.POSItem
 import class WooFoundation.CurrencyFormatter
 import class WooFoundation.CurrencySettings
+import enum Networking.OrderStatusEnum
```

Comment:
This model should not reach directly to the Networking layer, it would seem to indicate that part of the logic to update the order status should be moved to Yosemite rather than having it in the app layer. Or perhaps we could use a wrapper.